src/jupyter_contrib_nbextensions/nbextensions/toc2/main.css

```diff
@@ -38,13 +41,13 @@ padding-left: 20px;
 }
 
 #navigate_menu li  {
-  padding-left: 2px;
+  padding-left: 0px;
   clear: both;
   list-style-type: none;
 }
 
-#navigate_menu-level0  {padding-left: 10px;}
-#navigate_menu-level0 ul  {padding-left: 10px;}
+#navigate_menu-level0  {padding-left: 0px;}
+#navigate_menu-level0 ul  {padding-left: 0px;}
```

Comment:
although, any such modification should probably go into another, separate PR :sweat_smile: